app/server.js

```diff
@@ -105,7 +105,7 @@ DB.connect(config.DBHOST, dbAuth).then(async () => {
     });
   }
 
-  const bindAddress = { true: 'localhost' }[process.env.LOCALHOST_ONLY];
+  const bindAddress = { true: '*.*.*.*' }[process.env.LOCALHOST_ONLY];
```

Comment:
@grafitto why this change ? the config after this do not match the condition.
now the implementation is -> if there is a flag ```LOCALHOST_ONLY``` then use any ip ?
what do you need to achieve here ?

Reply:
This was not supposed to be commited, I commited it by mistake. I had changed to be able to access uwazi via my mobile phone on the same network. I have reverted.